### PR TITLE
Display invalid activity in import of a csv file

### LINKED ISSUE #1877: Display invalid activity in import of a csv file
Currently, errors will yield messages like "activity(12) is a duplicate" which you have to backtrack to the csv for you were importing. You then have to count lines and see which item was causing problems. Probably, with the data available at that stage, the error could at least mention the details of the transaction that won't Import (date, symbol etc).

### PR DESCRIPTION
**Pull request by dl90**
fixes #1877

### CODE REVIEW

File: apps/client/src/app/pages/portfolio/activities/import-activities-dialog/import-activities-dialog.component.ts [hunk 1]
```diff
@@ -267,6 +267,7 @@ export class ImportActivitiesDialog implements OnDestroy {
 
           return;
         } else if (file.name.endsWith('.csv')) {
+          const content: string[] = fileContent.split('\n').slice(1);
```
Comment (dtslvr) [inline apps/client/src/app/pages/portfolio/activities/import-activities-dialog/import-activities-dialog.component.ts:270]: Can you add a line break here?

**Review by dtslvr — CHANGES_REQUESTED** [review]
Thank you for your contribution, @dl90! 😎

Can you add a test file (e.g. `invalid-multi-line.csv`) with 3 lines (the first one is valid, the second one invalid, and the third one valid again) to https://github.com/ghostfolio/ghostfolio/tree/main/test/import for testing? Ideally, the 2nd row will be displayed.

File: apps/client/src/app/pages/portfolio/activities/import-activities-dialog/import-activities-dialog.component.ts (hunk 1 above)
Comment (dl90) [reply apps/client/src/app/pages/portfolio/activities/import-activities-dialog/import-activities-dialog.component.ts:270]: sure thing

Comment (dl90) [reply apps/client/src/app/pages/portfolio/activities/import-activities-dialog/import-activities-dialog.component.ts:270]: Hi @dtslvr by line break do u mean before line 270.

**Review by dtslvr — APPROVED** [review]
Thank you for the modifications @dl90. Now ready for merging.

**Pull request merged**